modules/base-commerce/sw/runtimeCaches.ext.js

```diff
@@ -0,0 +1,8 @@
+export default (self) => {
+  self.push({
```

Comment:
I dont like extensions here. Why not make RuntimeCache its own concept?

`my-module/serviceWorker/runtimeCache/_i.ts`

```
export default {
  urlPattern: /\/_i\//,
  handler: 'CacheFirst',
}
```